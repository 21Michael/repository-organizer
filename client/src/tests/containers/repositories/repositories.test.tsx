import React from "react";
import { render, cleanup } from "@testing-library/react";
import Repositories from "../../../containers/repositories/repositories";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";
import store from "../../../storeReduxToolkit/configStore";
import { Provider } from "react-redux";
import "mutationobserver-shim";

export default describe("Repositories: ", () => {
  afterEach(() => {
    cleanup();
  });

  test("repositories", () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Repositories />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
