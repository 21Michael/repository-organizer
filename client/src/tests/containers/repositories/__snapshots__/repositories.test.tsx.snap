// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repositories:  repositories 1`] = `
<div>
  <main
    class="main"
  >
    <div
      class="main__wrapper"
    >
      <div
        class="table__wrapper"
      >
        <table
          class="table"
        >
          <thead
            class="table__head"
          >
            <tr
              class="tr"
            >
              <th
                class="th"
              >
                name
              </th>
              <th
                class="th"
              >
                description
              </th>
              <th
                class="th"
              >
                stars
              </th>
              <th
                class="th"
              >
                creator name
              </th>
              <th
                class="th"
              >
                created at
              </th>
            </tr>
          </thead>
          <tbody
            class="table__body"
          />
        </table>
      </div>
    </div>
  </main>
</div>
`;
